api/commandValidator.js

```diff
@@ -1,32 +1,34 @@
 'use strict';
 
-const Joi = require('joi');
+const Joi = require('@hapi/joi');
```

Comment:
this can me reverted to `require('joi');` to avoid future changes.

Reply:
I think we should wait until this is resolved.
https://github.com/sideway/joi/issues/2411

Reply:
so what he's saying is  that ```The npm joi module is being resurrected and the current v17.1.1 has been published there (identical to the version available on @hapi/joi).```
current `@hapi/joi` & `joi` in npm are same